Specs/Renderer/AutomaticUniformSpec.js

```diff
@@ -7,6 +7,7 @@ import { OrthographicFrustum } from '../../Source/Cesium.js';
 import { OrthographicOffCenterFrustum } from '../../Source/Cesium.js';
 import { Pass } from '../../Source/Cesium.js';
 import { Texture } from '../../Source/Cesium.js';
+import { DirectionalLight } from '../../Source/Cesium.js';
```

Comment:
Fix include order

Reply:
I'm not actually what the new include ordering scheme is. Before when it was RequireJS they would be ordered by folder (`Renderer`, `Scene`, etc) and then alphabetically within that. Now with the ES6 changes should it be just alphabetic in the Specs?

Reply:
Opened https://github.com/AnalyticalGraphicsInc/cesium/issues/8512